core-ui/src/main/java/com/google/samples/apps/nowinandroid/core/ui/NewsResourceCard.kt

```diff
@@ -268,13 +268,19 @@ fun NewsResourceTopics(
     topics: List<Topic>,
     modifier: Modifier = Modifier
 ) {
+    var expandedTopic by remember { mutableStateOf<String?>(null) }
```

Comment:
```suggestion
    // Store the ID of the Topic which has its "following" menu expanded, if any. 
    // To avoid UI confusion, only one topic can have an expanded menu at a time. 
    var expandedTopicId by remember { mutableStateOf<String?>(null) }
```